### PR TITLE
[ui][fix] get rid of failing old mapping step

### PR DESCRIPTION
**Pull request by mcada**
//skip-ci

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by mcada** [comment]
@asmigala the step works with simple mappings, not with collections, should I remove it anyway?

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: ui-tests/src/test/java/io/syndesis/qe/steps/integrations/datamapper/DataMapperSteps.java [hunk 1]
```diff
@@ -151,8 +146,8 @@ public void combinePresentFielsWithAnother(String first, String first_pos,
         mapper.fillInput(inputElement, sec_pos);
 
         // Then fill in "TargetCombine" selector-input with "first_and_last_name" value
-//        inputElement = mapper.getElementByAlias("FirstTarget").shouldBe(visible);
-//        mapper.fillInputAndConfirm(inputElement, combined);
+        //        inputElement = mapper.getElementByAlias("FirstTarget").shouldBe(visible);
+        //        mapper.fillInputAndConfirm(inputElement, combined);
```
Comment (asmigala) [inline ui-tests/src/test/java/io/syndesis/qe/steps/integrations/datamapper/DataMapperSteps.java]: please remove commented code

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**